### ISSUE TITLE
Feature request: Objectification

### ISSUE THREAD

**Issue by RSAsterix** [nichtich/tkz-orm#8]
I would love to be able to objectify fact types. As it is now, I see no (obvious) way to do that.

I have made this beautiful mockup with MS Paint:
![image](https://cloud.githubusercontent.com/assets/7414682/26636151/23ad35fe-461c-11e7-8950-e4108e6b1036.png)

**Comment by camilstaps** [comment]
I agree. For the moment, you can hack this in by placing a `\entity` over a fact type and setting the `text width` and `opacity fill` (alternatively, first place the `\entity` and then the fact type, you probably don't need to set the `opacity fill` then).

This requires hard-coding the width of the fact type, unfortunately. It would be nice to have a more readable syntax and a backend that takes care of the scaling. Something like `\objectify (EntityName) at (FactName) {};` would already be nice (but, I don't think I will have time to look into scaling soon - perhaps nichtich does).

For now, the hack:

```tex
\documentclass{article}
\usepackage{tkz-orm}
\usetikzlibrary{positioning}
\begin{document}
\begin{tikzpicture}[orm]

\entity (A) at (0,0) {A};
\entity (B) at (4,0) {B};

\binary (link) at (2,0) {};
\plays (A) to (link);
\plays (B) to (link);

\entity [fill opacity=0, text width=1cm] (L) at (link) {};

\entity (C) at (2,-2) {C};

\vbinary (linkb) at (2,-1) {};
\plays (L) to (linkb);
\plays (C) to (linkb);

\end{tikzpicture}
\end{document}
```

![objectify](https://cloud.githubusercontent.com/assets/1202754/26636827/5e87f63c-460d-11e7-8032-7a1f5e3f5b51.png)
